index.html

```diff
@@ -7,5 +7,20 @@
     <link rel="stylesheet" href="styles.css" />
     <title>Lenz' Quizz App</title>
   </head>
-  <body></body>
+  <body>
+    <section class="card">
+      <img class="bookmark" src="/images/bookmark.gif" alt="bookmark image" />
+      <p class="question bigtext">
+        How many years did it take to build the Cologne cathedral?
+      </p>
+      <div class="showanswer shadow">
+        <p class="bigtext">Show answer.</p>
+      </div>
+      <div class="tags">
+        <div class="tag shadow"><p>#html</p></div>
+        <div class="tag shadow"><p>#css</p></div>
+        <div class="tag shadow"><p>#flexbox</p></div>
+      </div>
```

Comment:
Maybe `<ul>` with `display: flex;` is more reasonable than a `div` per item?

Reply:
I have no idea... ;)